UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/KeywordRecognitionHandler.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using UnityEngine.Events;
+using System.Collections.Generic;
+using System;
+using Microsoft.MixedReality.Toolkit;
+using Microsoft.MixedReality.Toolkit.Subsystems;
+
+public class KeywordRecognitionHandler : MonoBehaviour
```

Comment:
Please put this class under the `Microsoft.MixedReality.Toolkit.Examples` namespace